packages/openapi3/package.json

```diff
@@ -58,7 +58,7 @@
     "!dist/test/**"
   ],
   "dependencies": {
-    "@readme/openapi-parser": "~2.6.0",
+    "@apidevtools/swagger-parser": "~10.1.1",
```

Comment:
`@readme/openapi-parser` is a fork of `@apidevtools/swagger-parser` (even links to the same docs) that mainly claims to have more helpful error messages and some better handling of descriptions of `$refs`. However it uses some node modules in a way I couldn't get our bundler to play nice with (mainly `process`). Since I'm not calling the API that replaces `$refs` with the actual referenced objects, the only real downside is more verbose error message if Open API document validation fails.

Note: neither actually validate Open API 3.x schemas, only the document structure, so the impact really is pretty minimal